CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ The Product Changelog at **[piwik.org/changelog](http://piwik.org/changelog)** l
 * A new event `Db.getActionReferenceColumnsByTable` has been added in case a plugin defines a custom log table which references data to the log_action table 
 * A new JavaScript tracker method `getPiwikUrl` has been added to retrieve the URL of where the Piwik instance is located
 * A new JavaScript tracker method `getCurrentUrl` has been added to retrieve the current URL of the website. 
+* A new JavaScript tracker method `getNumTrackedPageViews` has been added to retrieve the number of tracked page views. 
```

Comment:
Won't that description be a bit confusing? Someone might think it makes it possible to retrieve the number of pageviews tracked within the visit. But actually it only returns the number of `trackPageView` calls done within the current loaded page.